sdk/src/manifest.rs

```diff
@@ -746,6 +782,7 @@ pub(crate) mod tests {
 
         assert_eq!(manifest.format(), "image/jpeg");
         assert_eq!(manifest.title(), Some("wc_embed_test.jpg"));
+        #[cfg(feature = "add_thumbnails")]
```

Comment:
Again, test the feature-disabled case?

Reply:
What should `manifest.thumbnail()` return when `add_thumbnails` feature is disabled?

Reply:
It returns the same things it does when enabled. The add_thumbnails feature only adds the ability to auto generate jpeg and png thumbnails. What manifest. thumbnail() returns depends on the test case. In some of these test cases, thumbnails will not be generated with add_thumbnails disabled, so we need to test for None in those cases only.